draft-ietf-quic-recovery.md

```diff
@@ -268,6 +268,13 @@ packets. While this increases network load, it is considered safe, since the
 sender will still reduce its sending rate exponentially under persistent
 congestion ({{pto}}).
 
+## Handshake packets are not special
+
+TCP treats the loss of SYN or SYN-ACK packet as persistent congestion and
+reduces the congestion window to one packet ({{?RFC5681}}). QUIC treats
+loss of a packet containing handshake data the same as other losses and
+reduces the congestion window.  When the PTO timer expires during the
+handshake, one or two packets are sent and the timer is doubled.
```

Comment:
I don't think that you need this last sentence.

Reply:
+1 to remove the last sentence... there are rules around PTO and handshake in 6.2.1 and 6.2.2, that should be enough I think. This would be repetitive, so if you think it bears repeating, then it's worth adding it, but I don't think we need it.